src/test/ui/rfcs/rfc-2396-target_feature-fn-ptr.rs

```diff
@@ -0,0 +1,9 @@
+#[target_feature(enable = "sse2")]
```

Comment:
Please move all relevant tests for the feature to `src/test/ui/rfc-2396-target_feature/`.